INSTALL.md

```diff
@@ -369,6 +370,19 @@ make -j$(nproc)
 sudo make install
 ```
 
+### sqlcipher
+
+If you are not using Fedora, skip this section, and go directly to installing [**toxcore**](#toxcore-dependencies).
```

Comment:
Thought you said  same place as FFMpeg  redirects which is ***toxcore*** not  #toxcore-compiling